pkg/codegen/go/gen.go

```diff
@@ -3124,15 +3126,9 @@ func generatePackageContextMap(tool string, pkg *schema.Package, goInfo GoPackag
 
 		if f.Inputs != nil {
 			pkg.names.Add(name + "Args")
-			if originalName != name {
```

Comment:
I believe this code was not doing what it appears to have been doing. Functions generate helper FArgs and FResult types but those types currently do not consult pkg.renamed tables, they continue generating as FArgs/FResult, but F the function name itself does consult those tables. So what this actually was doing, it would rename an unrelated type to collide with FArgs/FResult if there was one. Let's see what the downstream checks say, I expect there shouldn't be diffs based on this removal